inc/Loading/LoadListener.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2018 United States Government as represented by the
+// Administrator of the National Aeronautics and Space Administration.
+// All Rights Reserved.
+#ifndef LoadListener_h
```

Comment:
```suggestion
#ifndef PCOE_LOADLISTENER_H
```

To match style of other headers